### PR TITLE
Implement TallyResultIndicator

### PR DESCRIPTION
**Pull request by hochiw**
- Updated colorBar component to handle custom height with percentage display
- Implemented TallyResultIndicator with use case at ProposalDetail data section

## Screenshots 

| Desktop | Mobile |
|---------|---------|
|Screenshot 2022-06-22 at 2 07 27 PM|Screenshot 2022-06-22 at 2 07 34 PM|

refs oursky/likedao#126

### CODE REVIEW

File: react-app/src/components/common/ColorBar/ColorBar.tsx [hunk 1]
```diff
@@ -1,34 +1,90 @@
...
+  const { data, total, showPercentage } = props;
+  const [percentageEl, setPercentageEl] = useState<HTMLSpanElement | null>(
+    null
+  );
+
+  const percentage = useMemo(() => {
+    return data.value.div(total).shiftedBy(2).toFixed(1);
+  }, [data, total]);
+
+  useEffect(() => {
```
Comment (chihimng) [inline react-app/src/components/common/ColorBar/ColorBar.tsx:26]: do we need something like `useWindowSize`?

Comment (chihimng) [reply react-app/src/components/common/ColorBar/ColorBar.tsx:26]: how did we handle small percentage sections at the end?

**Review by elise-ng — CHANGES_REQUESTED** [review]
one question on responsive / small percentage sections

**Comment by hochiw** [comment]
@chihimng updated 🙏

**Review by elise-ng — APPROVED** [review]

**Pull request merged**